Formula/libgetdata.rb

```diff
@@ -12,14 +12,14 @@ class Libgetdata < Formula
   end
 
   option "with-fortran", "Build Fortran 77 bindings"
-  option "with-perl", "Build Perl binding"
```

Comment:
Maybe keep the option nonetheless so that it is clearer what type of Perl support this formula has to offer?